dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -475,6 +475,73 @@
 		</ItemGroup>
 	</Target>
 
+	<!-- Install & Run -->
+
+	<PropertyGroup>
+		<_MlaunchPath Condition="'$(_MlaunchPath)' == ''">$(_XamarinSdkRootDirectory)tools\bin\mlaunch</_MlaunchPath>
+	</PropertyGroup>
+
+	<Target Name="_PrepareRunMac" DependsOnTargets="_GenerateBundleName" Condition="'$(_PlatformName)' == 'macOS'">
+		<PropertyGroup Condition="'$(_PlatformName)' == 'macOS'">
+			<RunCommand>open</RunCommand>
+			<RunArguments>-a "$(_AppBundlePath)"</RunArguments>
+		</PropertyGroup>
```

Comment:
This is sort of redundant with https://github.com/xamarin/xamarin-macios/blob/59db0841419c9cbececea878ca74354ebf2a9bad/dotnet/Microsoft.macOS.Sdk/targets/Microsoft.macOS.Sdk.targets#L5-L8 (and doing `Condition="'$(_PlatformName)' == 'macOS'"` is likely unnecessary).

Reply:
🤦 (twice)